<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>473</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_op">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_so_de">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_op_so_serverName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_op_so_serverName">
           <property name="placeholderText">
            <string>Enter a servername, this name is used in Battlelog.</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_op_so_serverDescription">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="textEdit_op_so_serverDescription">
           <property name="placeholderText">
            <string>Enter a server description, this description is displayed on the loading screen.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_op_so_serverMessage">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_op_so_serverMessage">
           <property name="placeholderText">
            <string>Enter a massage here, this message is shown to player on join.</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>164</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>422</width>
           <height>168</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_so_co">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_so_co_gamePassword">
           <property name="text">
            <string>Game password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_so_co_gamePassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_so_co_adminPassword">
           <property name="text">
            <string>Admin password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_so_co_adminPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>198</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_so_co_serverTypeText">
           <property name="text">
            <string>Servertype</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_so_co_serverType">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_so_co_punkBuster">
           <property name="text">
            <string>PunkBuster</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_so_co_fairFight">
           <property name="text">
            <string>FairFight</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_so_co_maxSpectators">
             <property name="text">
              <string>Spectator limit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_so_co_maxSpectators"/>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_so_co_alwaysAllowSpectators">
           <property name="text">
            <string>Public spectators</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_so_co_maxPlayers">
             <property name="text">
              <string>Player limit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_so_co_maxPlayers"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_so_co_idleTimeout">
           <property name="text">
            <string>Disable idle kick</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_so_co_idleTimeout">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>86400</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_so_co_aggressiveJoin">
           <property name="text">
            <string>Aggresive join</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_so_co_commander">
           <property name="text">
            <string>Commander</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_so_co_idleBanRounds">
           <property name="text">
            <string>Disable idle ban</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_so_co_idleBanRounds"/>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>438</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_so_gp">
      <attribute name="title">
       <string>Gameplay</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="4">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_friendlyFire">
           <property name="text">
            <string>Friendly fire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_autoBalance">
           <property name="text">
            <string>Auto balance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_killCam">
           <property name="text">
            <string>Killcam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_miniMap">
           <property name="text">
            <string>Minimap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_miniMapSpotting">
           <property name="text">
            <string>Minimap spotting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_3dSpotting">
           <property name="text">
            <string>3D spotting</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_nameTag">
           <property name="text">
            <string>Show nametags</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_regenerateHealth">
           <property name="text">
            <string>Regenerate health</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_hud">
           <property name="text">
            <string>Show HUD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_onlySquadLeaderSpawn">
           <property name="text">
            <string>Spawn only on squad leader</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_vehicleSpawnAllowed">
           <property name="text">
            <string>Vehicles</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_hitIndicatorsEnabled">
           <property name="text">
            <string>Hit indicators</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_thirdPersonVehicleCameras">
           <property name="text">
            <string>Third person vehicle cameras</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_so_gp_forceReloadWholeMags">
           <property name="text">
            <string>Force reload of whole mags</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="3">
        <layout class="QFormLayout" name="formLayout_11">
         <item row="3" column="0">
          <widget class="QLabel" name="label_so_gp_playerRespawnTime">
           <property name="text">
            <string>Player respawn time (%)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_so_gp_ticketBleedRate">
           <property name="text">
            <string>Ticket bleed rate (%)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_so_gp_gameModeCounter">
           <property name="text">
            <string>Gamemode counter (%)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_so_gp_roundTimeLimit">
           <property name="text">
            <string>Round time limit (%)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_so_gp_roundLockdownCountdown">
           <property name="text">
            <string>Round lockdown countdown</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_so_gp_roundWarmupTimeout">
           <property name="text">
            <string>Round warmup timeout</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_so_gp_roundRestartPlayerCount">
           <property name="text">
            <string>Round restart player count</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_so_gp_roundStartPlayerCount">
           <property name="text">
            <string>Round start player count</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_playerRespawnTime">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_ticketBleedRate">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_gameModeCounter">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_roundTimeLimit">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_roundLockdownCountdown">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_roundWarmupTimeout">
           <property name="maximum">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_roundRestartPlayerCount">
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_roundStartPlayerCount">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_vehicleSpawnDelay">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_so_gp_vehicleSpawnDelay">
           <property name="text">
            <string>Vehicle spawn delay (%)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_soldierHealth">
           <property name="minimum">
            <number>60</number>
           </property>
           <property name="maximum">
            <number>125</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_so_gp_soldierHealth">
           <property name="text">
            <string>Solider health (%)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_so_gp_bulletDamage">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_so_gp_bulletDamage">
           <property name="text">
            <string>Bullet damage (%)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_so_gp_preset">
           <property name="text">
            <string>Game preset (Server side)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_so_gp_preset">
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Classic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hardcore</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Infantry</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_so_gp_presetLockPresetSetting">
           <property name="text">
            <string>Lock preset related values</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>272</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_so_gp_unlockMode">
           <property name="text">
            <string>Unlock unlocks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_so_gp_unlockMode">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Common</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Based on stats</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>113</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_so_tk">
      <attribute name="title">
       <string>Teamkilling</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>255</width>
         <height>95</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_8">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_so_tk_teamKillCountForKick"/>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_so_tk_teamKillCountForKick">
          <property name="text">
           <string>No teamkill counter limit</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_so_tk_teamKillKickForBan"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_so_tk_teamKillKickForBan">
          <property name="text">
           <string>No teamkill ban limit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_so_tk_teamKillValueForKick">
          <property name="text">
           <string>No teamkill value limit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_so_tk_teamKillValueForKick"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>310</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_9">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_so_tk_teamKillValueIncrease">
          <property name="text">
           <string>Value per teamkill</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_so_tk_teamKillValueDecreasePerSecond">
          <property name="text">
           <string>Teamkill value cooldown (per second)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_so_tk_teamKillValueDecreasePerSecond"/>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_so_tk_teamKillValueIncrease"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
